app/models/comment.rb

```diff
@@ -0,0 +1,2 @@
+class Comment < ApplicationRecord
+end
```

Comment:
- [ ] <b>REQUIRED:</b> Kindly add `associations` for comment.

```ruby
belongs_to :author, class_name: 'User'
belongs_to :post
```

Reply:
This change has been done perfectly